qlty-cli/src/commands/coverage/complete.rs

```diff
@@ -0,0 +1,403 @@
+use crate::{CommandError, CommandSuccess};
+use anyhow::{anyhow, Context};
+use anyhow::{bail, Result};
+use clap::Args;
+use console::style;
+use git2::Repository;
+use qlty_cloud::Client as QltyClient;
+use qlty_config::version::LONG_VERSION;
+use qlty_config::{QltyConfig, Workspace};
+use qlty_coverage::ci::{GitHub, CI};
+use qlty_coverage::eprintln_unless;
+use qlty_coverage::publish::{Plan, Planner, Settings};
+use regex::Regex;
+use serde_json::Value;
+use std::path::PathBuf;
+use std::time::Instant;
+use tracing::debug;
+
+const COVERAGE_TOKEN_WORKSPACE_PREFIX: &str = "qltcw_";
+const COVERAGE_TOKEN_PROJECT_PREFIX: &str = "qltcp_";
+const OIDC_REGEX: &str = r"^([a-zA-Z0-9\-_]+)\.([a-zA-Z0-9\-_]+)\.([a-zA-Z0-9\-_]+)$";
+const LEGACY_API_URL: &str = "https://qlty.sh/api";
+
+#[derive(Debug, Args, Default)]
+pub struct Complete {
+    #[arg(long)]
+    pub tag: Option<String>,
+
+    #[arg(long)]
+    /// Override the branch from the CI environment
+    pub override_branch: Option<String>,
+
+    #[arg(long)]
+    /// Override the commit SHA from the CI environment
+    pub override_commit_sha: Option<String>,
+
+    #[arg(long)]
+    /// Override the pull request number from the CI environment
+    pub override_pr_number: Option<String>,
```

Comment:
I would add override_build_id to keep it in sync with publish. L

Despite build ID not being required, it is useful, and we would prefer it be sent with every API call